plugins/woocommerce-beta-tester/src/tools/commands/set-wccom-request-errors.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { SelectControl } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { STORE_KEY } from '../data/constants';
+
+export const UPDATE_WCCOM_REQUEST_ERRORS_MODE = 'updateWccomRequestErrorsMode';
+
+const OPTIONS = [
+	{ label: 'Timeout', value: 'timeout' },
```

Comment:
A bit more verbose helps I think
```suggestion
	{ label: 'Timeout after 6 seconds', value: 'timeout' },
```